alacritty.yml

```diff
@@ -518,12 +520,14 @@ key_bindings:
   - { key: Right,    mods: Alt,     chars: "\x1b[1;3C"                       }
   - { key: Right,                   chars: "\x1b[C",        mode: ~AppCursor }
   - { key: Right,                   chars: "\x1bOC",        mode: AppCursor  }
-  - { key: Up,       mods: Shift,   chars: "\x1b[1;2A"                       }
+  - { key: Up,       mods: Shift,   action: ScrollLineUp,   mode: ~Alt       }
```

Comment:
We can *not* bind these by default, since they are escape sequences which can be read by terminal applications.

Reply:
Ah, okay.  Fixed.  (It's <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Up / Down</kbd> now.)

Reply:
This is not standard functionality. While some people might be interested in scrolling line-by-line (which is fine), it should not be bound by default.

Reply:
Sorry for the delayed response.

The following terminal emulators bind keys to line-by-line scrolling by default:

+ xfce4-terminal binds Ctrl + Shift + Up/Down
+ LXTerminal binds Ctrl + Shift + Up/Down
+ GNOME Terminal binds Ctrl + Shift + Up/Down
+ kitty binds Ctrl + Shift + Up/Down
+ tilda binds Ctrl + Shift + Up/Down
+ Terminator binds Ctrl + Shift + Alt + Up/Down
+ Konsole binds Shift + Up/Down

The only terminal emulator I tested that didn't have a keybinding for line-by-line scrolling is XTerm.  I don't think having a keybinding for line-by-line scrolling by default is uncommon.  I think users will appreciate if something like this works out of the box.  That being said, if you still want to remove the keybinding, then I'll remove it.

Reply:
URxvt also does not set any binding by default.

I think this topic is too opinionated to find a good default binding which would still be easy to trigger without breaking workflows of some people.

Generally it shouldn't be a huge issue to set these up yourself if you're interested in them.

Reply:
Fair enough; fixed.  :)